app/car.py

```diff
@@ -0,0 +1,7 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Car:
+    brand: str
+    fuel_consumption: float

```

Comment:
Consider adding a method to calculate or return the fuel consumption based on the task requirements. This might be necessary if the task involves operations related to fuel consumption.